student.rb

```diff
@@ -0,0 +1,15 @@
+require_relative('./person')
+
+class Student < Person
+  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
+    @classroom = classroom
+  end
+
+  def play_hooky
+    '¯(ツ)/¯'
+  end
+end
+
+student1 = Student.new(12, 4, 'oswald')
+puts student1.name
```

Comment:
- [ ] please make sure to remove any code intended for testing purposes, this will make your code look unorganised and harder to read.